lib/hammer_cli_foreman/subnet.rb

```diff
@@ -9,6 +9,7 @@ class ListCommand < HammerCLIForeman::ListCommand
       output do
         field :id, _("Id")
         field :name, _("Name")
+        field :description, _("Description")
```

Comment:
just a general query, if i use this version of hammer against an older version of foreman, does this work as expected? e.g. the description header is not shown or its shown but its value is always empty?